libs/jsonschema/extension.go

```diff
@@ -12,6 +12,10 @@ type Extension struct {
 	// that do have an order defined.
 	Order *int `json:"order,omitempty"`
 
+	// The message to print after the template is successfully initalized
+	// In line with our ymls, we use snake_case here.
```

Comment:
What is specific about this field, re: snake case?

Reply:
This was our very first extension. And snake_case is pretty special since other JSON schema properties don't use it: https://github.com/databricks/cli/blob/f24e8dff95ef8299fb63bd3900c44cbe3f093468/libs/jsonschema/schema.go

But we now also have `pattern_match_failure_message` that uses the same casing. I can remove the remark.